nio_cli/commands/config.py

```diff
@@ -26,8 +28,61 @@ def config_project(name='.'):
                 tmp.write('PK_TOKEN={}\n'.format(pk_token))
             else:
                 tmp.write(line)
-        os.remove(env_location)
-        os.rename(tmp.name, env_location)
+        os.remove(conf_location)
+        os.rename(tmp.name, conf_location)
+    secure = input('Optional secure instance configuration [Y/N]: ')
+    if secure.lower() == 'y':
+        config_ssl(name, conf_location)
+
+
+def config_ssl(name, conf_location):
+
+    v = ssl.OPENSSL_VERSION.split()[0]
+    if v != 'OpenSSL':
+        print('No OpenSSL installation detected')
```

Comment:
Couple things:
1. Do they need openssl if they are just going to provide their own path to their certs? I wouldn't think so but not 100% sure.
2. We should make it more clear that the instance is still configured but that no certs were installed. And that if they want to add certs they should install openssl. Maybe something like this:
```
No OpenSSL installation detected. Your instance has still be configured but no certs were installed. To install certificates install OpenSSL and re-run "nio configure"
```